README.md

```diff
@@ -281,6 +281,10 @@ build.targets = []
 # Verbose printout when building.
 build.verbose = false
 
+# Additional ``build-system.requires``. Intended to be used in combination with
+# ``overrides``.
+build.build-requires = []
```

Comment:
Should this be `build.requires` instead of `build.build-requires`?

Reply:
Probably. As long as it can't be confused with `dependencies` it should be fine to simplify it.